In [1]:
import numpy as np
import pandas as pd
from scipy import stats

In [2]:
data=pd.read_csv("BuyerRatio.csv")
data.head()

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [3]:
Males = data.iloc[0,1:]
Females = data.iloc[1,1:]

In [4]:
data1 = [Males,Females]
print(data1)

[East      50
West     142
North    131
South     70
Name: 0, dtype: object, East      435
West     1523
North    1356
South     750
Name: 1, dtype: object]


In [5]:
result = stats.chi2_contingency(data1)
print(result)

(1.595945538661058, 0.6603094907091882, 3, array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
       [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]]))


In [6]:
stat_value = result[0]
pi_value = result[1]
df = result[2]
stat_value, pi_value, df

(1.595945538661058, 0.6603094907091882, 3)

Ans) As p-value(0.6603) is greater than alpha significance(0.05) we can conclude that - "We fail to reject null hypothesis i.e. All proportions are equal."